tools/choco/tinytex/tinytex.nuspec

```diff
@@ -11,7 +11,7 @@
     <projectUrl>https://yihui.org/tinytex/</projectUrl>
     <iconUrl>https://yihui.org/images/logo-tinytex.png</iconUrl>
     <copyright>Copyright 2020 Naveen M K,Yihui Xie</copyright>
```

Comment:
From https://tug.org/texlive/LICENSE.TL, I feel that you and I shouldn't be the copyright holder:

> You absolutely may *not* place your own copyright on the entire
distribution, since it is not your work.  Statements such as "all rights
reserved" and "may not be reproduced" are especially reprehensible,
since they are antithetical to the free software principles under which
TeX Live is produced.

Perhaps use "TeX Live Developers"?

Reply:
I will fix it soon.

Reply:
I will fix it soon.